application-tour-ui/src/main/resources/TourCode/TourJson.xml

```diff
@@ -132,6 +147,5 @@
   #set ($discard = $response.setContentType('application/json'))
   $jsontool.serialize({'tours': $tours})
 #end
-{{/velocity}}
-</content>
+{{/velocity}}</content>
```

Comment:
I agree with your comment, these changes seem like they shouldn't be in the diff, but how do I simply revert them? I develop locally on my wiki, then export the XARs I need and run xar:format on top of it - but these changes sneak in. How can I avoid it?